public/static/locales/en-US/attendance.json

```diff
@@ -59,6 +59,7 @@
     "absenceType": {
       "sick": "Sick",
       "covid": "COVID-19",
+      "activity":"Activity",
```

Comment:
Add a space between `:` and `"Activity"`. It’s good practice to have Prettier set up on your code editor, and regularly formatting the file with `Shift`+`Alt`+`F`.

Same in the Thai file as well.